test/jest/security.test.tsx

```diff
@@ -268,11 +282,20 @@ describe('<Security />', () => {
 
     it('should render error if oktaAuth props is not provided', () => {
       const wrapper = mount(
-        <Security oktaAuth={null}>
+        <Security oktaAuth={null} navigate={navigate}>
           <MyComponent />
         </Security>
       );
       expect(wrapper.find(Security).html()).toBe('<p>AuthSdkError: No oktaAuth instance passed to Security Component.</p>');
     });
+
+    it('should render error if navigate prop is not provided', () => {
```

Comment:
I've not looked into this, but is there a reason we're rendering an error here instead of using prop-types?

Reply:
I've implemented `navigate` prop checking same way as for `oktaAuth` prop, for consistency.